### PR TITLE
Remove explicit hints

### PR DESCRIPTION
**Pull request by sealocal**
This PR removes some additions from PR #78 and makes some misc. style improvements.

### CODE REVIEW

File: chapter_4_rails_theming,_continuous_integration,_and_deployment/deploy_to_heroku.md [hunk 1]
```diff
@@ -9,30 +9,28 @@ https://devcenter.heroku.com/articles/getting-started-with-rails4
 
 * One really important step that is left out of that article is that you need to tell Rails that it is ok to serve static assets, which Rails 4 will not do by default. If you don't do that, your website images will seem to vanish over time.
 
-* Find these settings in the Production Environment config file (`config/environments/production.rb`), which is used on Heroku, and set them to true - `config.serve_static_files` and `config.assets.digest`
+    Find that setting in the Production Environment config file (`config/environments/production.rb`), which is used by Heroku. Change that value to `true`.
```
Comment (sealocal) [inline chapter_4_rails_theming,_continuous_integration,_and_deployment/deploy_to_heroku.md]: `config.assets.digest` is set to true by default.  I think these hints are too explicit given the explanation above.

File: chapter_4_rails_theming,_continuous_integration,_and_deployment/set_up_a_continuous_integration_server_for_your_repo.md [hunk 2]
```diff
@@ -5,23 +5,24 @@ Get your tests running in the cloud, and put a nice green badge on your README o
...
-* [Introducing Continuous Deployment to Heroku](http://about.travis-ci.org/blog/2013-07-09-introducing-continuous-deployment-to-heroku/)
+  Follow the instructions in the guide:
+[Introducing Continuous Deployment to Heroku](http://about.travis-ci.org/blog/2013-07-09-introducing-continuous-deployment-to-heroku/).
 
 * `gem install travis`
 
 * `travis login`
 
-* `travis setup heroku` - this will automatically find your Heroku API key, and encrypt it in your `.travis.yml` file.
+* `travis setup heroku`
```
Comment (sealocal) [inline chapter_4_rails_theming,_continuous_integration,_and_deployment/set_up_a_continuous_integration_server_for_your_repo.md]: Removing some explicit explanation from this hint. `travis setup heroku` may be too explicit alone.  Opinions?

File: chapter_4_rails_theming,_continuous_integration,_and_deployment/deploy_to_heroku.md [hunk 3]
```diff
@@ -44,9 +42,7 @@ https://www.codefellows.org/blog/three-battle-tested-ways-to-install-postgresql
 
       $ heroku run rake db:migrate
 
-* If you're using Google Fonts and run into issues with it loading, either change the protocol from `http://fonts...` to `https://fonts...`, or leave the protocol out and just do `//fonts...`
-
```
Comment (sealocal) [inline chapter_4_rails_theming,_continuous_integration,_and_deployment/deploy_to_heroku.md]: This tip is specific to Google, and the web in general.  This guide is for Heroku. Google Fonts has not been recommended in this book yet, so I think it's irrelevant at this time.

**Comment by sealocal** [comment]
This PR removes some additions from PR #78 and makes some misc. style improvements.

File: chapter_4_rails_theming,_continuous_integration,_and_deployment/set_up_a_continuous_integration_server_for_your_repo.md (hunk 2 above)
Comment (brookr) [inline chapter_4_rails_theming,_continuous_integration,_and_deployment/set_up_a_continuous_integration_server_for_your_repo.md]: The danger is that people will run the commands without understanding what they do.

File: chapter_4_rails_theming,_continuous_integration,_and_deployment/deploy_to_heroku.md (hunk 3 above)
Comment (brookr) [inline chapter_4_rails_theming,_continuous_integration,_and_deployment/deploy_to_heroku.md]: Some people may be using Google fonts in a third-party Zurb template.

File: chapter_4_rails_theming,_continuous_integration,_and_deployment/set_up_a_continuous_integration_server_for_your_repo.md (hunk 2 above)
Comment (sealocal) [inline chapter_4_rails_theming,_continuous_integration,_and_deployment/set_up_a_continuous_integration_server_for_your_repo.md]: So, do you want to include the tip with explanation?

File: chapter_4_rails_theming,_continuous_integration,_and_deployment/deploy_to_heroku.md (hunk 3 above)
Comment (sealocal) [inline chapter_4_rails_theming,_continuous_integration,_and_deployment/deploy_to_heroku.md]: That makes sense. I'll add it back.

File: chapter_4_rails_theming,_continuous_integration,_and_deployment/set_up_a_continuous_integration_server_for_your_repo.md (hunk 2 above)
Comment (sealocal) [inline chapter_4_rails_theming,_continuous_integration,_and_deployment/set_up_a_continuous_integration_server_for_your_repo.md]: Or remove that command completely?

Comment (brookr) [inline chapter_4_rails_theming,_continuous_integration,_and_deployment/set_up_a_continuous_integration_server_for_your_repo.md]: Yeah, you can remove it completely, it's explained in the blog post.

**Pull request merged**